src/olympia/landfill/serializers.py

```diff
@@ -42,8 +43,10 @@ class GenerateAddonsSerializer(serializers.Serializer):
     count = serializers.IntegerField(default=10)
 
     def __init__(self):
-        self.fxa_email = os.environ['UITEST_FXA_EMAIL']
-        self.fxa_password = os.environ['UITEST_FXA_PASSWORD']
+        self.fxa_email = os.environ.get(
+            'UITEST_FXA_EMAIL', 'uitest-%s@restmail.net' % uuid.uuid4())
+        self.fxa_password = os.environ.get(
+            'UITEST_FXA_PASSWORD', 'uitester')
```

Comment:
Note: I immensely dislike how the landfill code depends on uitest specific variables. That really shouldn't be the case, but that's a larger fix.

Reply:
That code was initially written only for ui-tests and wasn't used to generate our dummy data during local setup.

So yeah, there are most certainly better ways to split this up and make it more reusable.